### ISSUE TITLE
Validation loss missing - U-Net_2D

### ISSUE THREAD

**Issue by xcarmengrandix** [HenriquesLab/ZeroCostDL4Mic#146]
Hello, 

I'm trying to train the network using images I manually labelled on U-Net_2D. At the moment I have a really small dataset but everything works correctly when running the script. The only issue I find is that when training I get _nan_ in the val_loss values:

Epoch 1/200
5/5 [==============================] - 2s 454ms/step - loss: 1.0597 - val_loss: nan

and indeed, when plotting the training errors vs. epoch number the orange line of the validation loss is missing. Am I missing something in the training step?

Thank you in advance for your help!

Carmen

**Comment by guijacquemet** [comment]
Great, keep us updated !
Cheers
Guillaume

**Comment by xcarmengrandix** [comment]
> Great, keep us updated ! Cheers Guillaume

Hi Guillaume,

I increased the polling steps and the percentage validation  under the training parameters and now I get the val_loss values too! ;)
